src/Cafe/HW/Latte/Renderer/Vulkan/VulkanPipelineStableCache.cpp

```diff
@@ -46,8 +46,11 @@ uint32 VulkanPipelineStableCache::BeginLoading(uint64 cacheTitleId)
 	m_numCompilationThreads = std::clamp(cpuCoreCount, 1u, 8u);
 	if (g_renderer->GetVendor() == GfxVendor::Nvidia)
```

Comment:
If instead of saving the driver version you were simply saving a "m_disableMultithreadedCompilation" flag you could move out the vendor/driver-specific logic out and this file and simply this block further to
```c
if(VulkanRenderer::GetInstance()->GetDisableMultithreadedCompilation())
    m_numCompilationThreads = 1;
```

Reply:
I agree with this suggestion. The driver version format differs by vendor so there isn't actually a nice way to have a singular `GetDriverVersion` function that makes sense.
The `disableMultithreadedCompilation` flag can be put into the `m_featureControl` member struct and it's overkill to wrap it in a getter. We don't do it with other options like `debugMarkersSupported`

Reply:
Made this use GetDisableMultithreadedCompilation() as a getter, because m_featureControl was private